util/config/schema.py

```diff
@@ -702,6 +702,16 @@
             "description": "The number of seconds between indexing intervals in the security scanner. Defaults to 30.",
             "x-example": 30,
         },
+        "SECURITY_SCANNER_V4_SIGN_JWT": {
```

Comment:
Only add a single field: `SECURITY_SCANNER_V4_JWT_PSK` and make the decision to sign/verify requests with JWT based on the presence or absence of that field.